src/collection-preferences/content-display/index.tsx

```diff
@@ -98,6 +113,33 @@ export default function ContentDisplayPreference({
       <p className={getClassName('description')} id={descriptionId}>
         {i18n('contentDisplayPreference.description', description)}
       </p>
+
+      {/* Filter input */}
+      {searchableColumns && (
+        <div className={getClassName('text-filter')}>
+          <InternalTextFilter
+            filteringText={columnFilteringText}
```

Comment:
I believe these messages can be standard as the target resource is always columns. In that case there is a value of using the built-in i18n mechanism, see e.g. the 'contentDisplayPreference.description' from a few lines above.

For that to work we'd need to provide translations but as a POC it is fine to modify the messages types files and en translations under src/i18n.